### PR TITLE
fix for issue #68: send `embed` and `components` only if filled

### PR DESCRIPTION
**Pull request by deto1986**
note: discord have no documented `embed` attribute, it is called `embeds` and should be send as an array. Maybe in the older days it was called `embed`. I go with the current documentation here, see https://discord.com/developers/docs/resources/channel#create-message

### CODE REVIEW

**Comment by dpslwk** [comment]
api v10 takes the newer array "embeds", 
not sure if the default api (v6) will,
since a version is not specified v6 will be used as per https://discord.com/developers/docs/reference

**Comment by deto1986** [comment]
I have tested it, if nothing is specified the newer array "embeds" is working correctly.

File: src/DiscordChannel.php [hunk 1]
```diff
@@ -31,16 +31,24 @@ public function __construct(Discord $discord)
      */
     public function send($notifiable, Notification $notification)
     {
-        if (! $channel = $notifiable->routeNotificationFor('discord', $notification)) {
+        if (!$channel = $notifiable->routeNotificationFor('discord', $notification)) {
```
Comment (dpslwk) [inline src/DiscordChannel.php:34]: suspect this change will fail StyleCI check since the default in Laravel is with the space

Comment (deto1986) [reply src/DiscordChannel.php]: I have fixed it with another commit. I personally don't use StyleCI nor I have an integration in my IDE for that.

**Comment by winkcler** [comment]
I'm waiting for someone to approve this pull request.

**Comment by hashimotoryoh** [comment]
Thx for fixing. I'm waiting for merging this pr.

**Comment by winkcler** [comment]
Thx for fixing. I'm waiting for merging this pr.

**Comment by deto1986** [comment]
For everyone who's waiting for this PR, maybe you can implement a workaround for your stuff. With a filled `embed` you don't run into this error. Maybe you can fill a `embed` with some useful data as a workaround until this PR is merged.

**Comment by jee7** [comment]
When could this PR be merged? If it's estimated to take like a couple of weeks, then implementing the workaround seems like a good choice (although, I don't have anything useful to embed... we just have markdown messages as in #68). But if it's likely just a few more days, then I'd rather wait for it.

**Comment by LukasJK** [comment]
Can this please be merged?

**Comment by zeerakahmad** [comment]
Why has this not been merged? Can someone please escalate and merge. 🙏

**Comment by deto1986** [comment]
I have send an email to the maintainer @atymic  I hope he responds quickly. Sadly only the maintainer can approve PRs.

I have also done some research, maybe @codyphobe or @arcdigital can merge this PR too.

**Comment by arcdigital** [comment]
I can take a look later today!

**Comment by atymic** [comment]
Sorry guys, I didn't see the emails from github. Thanks @deto1986 for emailing me.

If anyone is interested in taking over maintenance of this channel please comment and we can have a chat.

**Comment by arcdigital** [comment]
> Sorry guys, I didn't see the emails from github. Thanks @deto1986 for emailing me.
> 
> If anyone is interested in taking over maintenance of this channel please comment and we can have a chat.

I'm interested, I use this channel quite often.

**Pull request merged**